pandas/core/groupby/groupby.py

```diff
@@ -3549,13 +3549,11 @@ def _aggregate_multiple_funcs(self, arg, _level):
                 obj._selection = name
             results[name] = obj.aggregate(func)
 
-        if isinstance(list(compat.itervalues(results))[0],
-                      DataFrame):
-
+        if any(isinstance(x, DataFrame) for x in compat.itervalues(results)):
             # let higher level handle
             if _level:
                 return results
-            return list(compat.itervalues(results))[0]
```

Comment:
There was no test coverage for this line and I can't feasibly think how this would be useful, hence the deletion